app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -194,8 +194,8 @@ class Contact extends React.Component {
           <Card.Header className="h5">Monitoree Contact Information</Card.Header>
           <Card.Body>
             <Form>
-              <Form.Row className="pb-3">
-                <Form.Group as={Col} md="8" controlId="preferred_contact_method">
+              <Form.Row className="mb-2">
+                <Form.Group as={Col} lg="12" xl="10" controlId="preferred_contact_method">
```

Comment:
I originally did it because the inputs were very very large and looked odd.  I fixed the warning issues though and added some resize styling that wasn't there before: https://github.com/SaraAlert/SaraAlert/pull/941/commits/7663a8a00ea417ec1925dd05ad750069fd46b304

![Screen Shot 2021-06-17 at 2 24 42 PM](https://user-images.githubusercontent.com/35042815/122454338-49bc4600-cf79-11eb-8784-67860d8e689f.png)

![Screen Shot 2021-06-17 at 2 25 10 PM](https://user-images.githubusercontent.com/35042815/122454341-4a54dc80-cf79-11eb-8ba1-e077fca06819.png)

![Screen Shot 2021-06-17 at 2 25 17 PM](https://user-images.githubusercontent.com/35042815/122454342-4a54dc80-cf79-11eb-830c-866e02bba5a6.png)

Reply:
the medium and small screen sizes look fine but can we consult the PO team about the `xl="10"`? it doesn't look quite right, or at least maybe make it `xl=8` so it's somewhat consistent with the other enrollment steps?

Reply:
I changed that in the latest version with the banner, so no worries there